src/hooks/index.js

```diff
@@ -120,7 +121,10 @@ function checkLoadedModules(logger) {
 }
 
 function activate(agent) {
-
+  if (activated) {
```

Comment:
Oh - it was being activated multiple times in an intermediate form of a test. I've since refactored that test so this does not apply anymore.